[2024-08-12T13:20:00.649+0500] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-12T13:20:00.682+0500] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train.init scheduled__2024-08-11T01:00:00+00:00 [queued]>
[2024-08-12T13:20:00.709+0500] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train.init scheduled__2024-08-11T01:00:00+00:00 [queued]>
[2024-08-12T13:20:00.709+0500] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-12T13:20:00.727+0500] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): init> on 2024-08-11 01:00:00+00:00
[2024-08-12T13:20:00.742+0500] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'train', 'init', 'scheduled__2024-08-11T01:00:00+00:00', '--job-id', '236', '--raw', '--subdir', 'DAGS_FOLDER/train_dag.py', '--cfg-path', '/tmp/tmpozy3hpy4']
[2024-08-12T13:20:00.744+0500] {standard_task_runner.py:91} INFO - Job 236: Subtask init
[2024-08-12T13:20:00.744+0500] {logging_mixin.py:188} WARNING - /home/nicolaspc/anaconda3/envs/mlops_env/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=14286) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-12T13:20:00.745+0500] {standard_task_runner.py:64} INFO - Started process 14451 to run task
[2024-08-12T13:20:00.804+0500] {task_command.py:426} INFO - Running <TaskInstance: train.init scheduled__2024-08-11T01:00:00+00:00 [running]> on host nicolaspc-HP-Pavilion-Laptop-15-eh1xxx
[2024-08-12T13:20:00.916+0500] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nicolas Pal' AIRFLOW_CTX_DAG_ID='train' AIRFLOW_CTX_TASK_ID='init' AIRFLOW_CTX_EXECUTION_DATE='2024-08-11T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-11T01:00:00+00:00'
[2024-08-12T13:20:00.917+0500] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-12T13:20:00.952+0500] {base.py:84} INFO - Using connection ID 'aws_default' for task execution.
[2024-08-12T13:20:00.953+0500] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-08-12T13:20:02.970+0500] {train_dag.py:84} INFO - Data download finished.
[2024-08-12T13:20:02.971+0500] {python.py:237} INFO - Done. Returned value was: None
[2024-08-12T13:20:02.972+0500] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-12T13:20:02.982+0500] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=train, task_id=init, run_id=scheduled__2024-08-11T01:00:00+00:00, execution_date=20240811T010000, start_date=20240812T082000, end_date=20240812T082002
[2024-08-12T13:20:03.010+0500] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-12T13:20:03.034+0500] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-12T13:20:03.037+0500] {local_task_job_runner.py:222} INFO - ::endgroup::
